web/templates/pages/tasks.html

```diff
@@ -0,0 +1,40 @@
+{% extends 'base.html' %}
+
+{% block title %}All CodeCatz Tasks{% endblock %}
+
+{% block content%}
+
+	<header class="row">
+		<div class="col-lg-6 col-sm-7">
+		      <h1>All CodeCatz Tasks</h1>
+		</div>
+	</header>
+
+	{% if task_list %}
+
+		<div class="row">
+
+			{% for task in task_list %}
+
+				<div class="col-md-3 col-xs-6">
+					<h3><strong>Task title: </strong>{{ task.title }}</h3>
+					<p><strong>Assigned to: </strong> {{ task.assigned_id }}</p>
```

Comment:
you're "a bit" inconsistent with your use of spaces :wink:

remove the following spaces:
* [line 21](https://github.com/CodeCatz/TrackCat/pull/155/files#diff-b51cc89427948a9e577a883015e9bc9eR21): "\</strong> {{"
* [line 22](https://github.com/CodeCatz/TrackCat/pull/155/files#diff-b51cc89427948a9e577a883015e9bc9eR22): "href= \"\{", "> {{" and "}} \</a>"
* [line 24](https://github.com/CodeCatz/TrackCat/pull/155/files#diff-b51cc89427948a9e577a883015e9bc9eR24): "\</strong> {{" and "}} \</p>"
* [line 26](https://github.com/CodeCatz/TrackCat/pull/155/files#diff-b51cc89427948a9e577a883015e9bc9eR26): "\</strong> {{" and "}} \</p>"
* [line 35](https://github.com/CodeCatz/TrackCat/pull/155/files#diff-b51cc89427948a9e577a883015e9bc9eR35): "\<p> There"